test/packages/good_v3/data_stream/foo/fields/some_fields.yml

```diff
@@ -89,3 +89,8 @@
 - name: empty_expected_values
   type: keyword
   expected_values: []
+- name: some_counted_keyword
+  type: counted_keyword
```

Comment:
I have noticed that this type is incompatible with synthetic source. Regarding this change this is fine, but just in case you want to use it in some APM data streams that use it.
```
field [event.counted] of type [counted_keyword] doesn't support synthetic source
```

Reply:
We we'll use this field type on the data stream storing traces, which does not use synthetic source. But thanks for the heads-up!